### PR TITLE
Added Presentation as a possible domain for termsOfUse to the formal vocabulary

### PR DESCRIPTION
**Pull request by iherman**
Syncing with the latest change of the v2 context

### CODE REVIEW

**Comment by iherman** [comment]
(Should be merged only when the automatic vocabulary generation is operational.)

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
> (Should be merged only when the automatic vocabulary generation is operational.)

Automatic vocabulary generation is operational.

**Comment by iherman** [comment]
@msporny I presume that you have the right and honor to merge PR-s... and I am not supposed to do it

**Comment by msporny** [comment]
> @msporny I presume that you have the right and honor to merge PR-s... and I am not supposed to do it

Yes, I'm waiting for more people to see it/review it before merging. I almost merged it, noting that it was an "Editorial" change based on a previous PR that went in... but then thought I'd leave it floating out there for a bit because sometimes these sorts of PRs trigger people to make comments about other things we should be looking out for (or to assert that we're making a mistake for X or Y reason). Unless you object, I was going to give it the standard 7 day delay before merging.

File: vocab/credentials/v2/vocabulary.yml
Comment (TallTed) [inline vocab/credentials/v2/vocabulary.yml:142]:
```suggestion
      If specified, the value of the optional `termsOfUse` property MUST specify one or more terms of use policies under which the issuer issued the credential or presentation. Each `termsOfUse` value MUST specify its type (for example, `IssuerPolicy`) and MAY specify its instance `id`. The precise content of each term of use is determined by the specific `TermsOfUse` type definition. If the recipient (a holder or verifier) violates the specified terms of use, the responsibility is their own, and such violation may incur legal liability.
```

Comment (iherman) [reply vocab/credentials/v2/vocabulary.yml:142]: I am personally ok with the change, @TallTed, thanks. But I would prefer a domain expert to give the thumbs up before taking this over. My only slight reservation is that this document is, formally, not a recommendation, so I am not sure whether a _MAY_ is appropriate instead of a "may".

Comment (iherman) [reply vocab/credentials/v2/vocabulary.yml:142]: (I realize that _MUST_ was already part of the text, b.t.w., so my question is general, and not closely related to your change proposal.)

Comment (TallTed) [reply vocab/credentials/v2/vocabulary.yml:142]: Given that the original text was `MUST ... optionally`, I can't imagine any domain expert having an issue with the `MAY`, though they (and I, on rereading) may prefer `MAY additionally`.

**Comment by jandrieu** [comment]
I'm confused. Doesn't @context already solve this problem?

**Review by brentzundel — APPROVED** [review]

**Comment by TallTed** [comment]
@jandrieu — How does "`@context` already solve this problem"? And what do you see as "this problem" that `@context` somehow solves?

(Also, please edit https://github.com/w3c/vc-data-model/pull/1013#issuecomment-1403861073 and code fence `@context`...)

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-01-25-vcwg#section4-4) on 2023-01-25

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.4. Added Presentation as a possible domain for termsOfUse to the formal vocabulary (pr vc-data-model#1013)

_See github pull request [vc-data-model#1013](https://github.com/w3c/vc-data-model/pull/1013)._




**Manu Sporny:** straightforward.  
… terms of use PR, syncing with latest changes.  
… follow on, mostly editorial.  

</details>

**Comment by iherman** [comment]
> I'm confused. Doesn't `@context` already solve this problem?

No. `@context` is not a vocabulary specification, it is a _mapping_ from JSON to a vocabulary that is to be specified separately. The vocabulary specification, for example, defines constraints that RDF tools can check, draw conclusions of, etc, all the features that are not in the realm of a `@context`.

In this specific case, the `@context` file does define the mapping for a `termOfUse` property when used in a presentation but, in fact, this conflicts with the current status of the vocabulary which defines `termOfUse` as being used on credentials only. An RDF reasoner may then deduce, in a specific case, that a presentation is in fact a credential...

The proposed PR takes care of this.

File: vocab/credentials/v2/vocabulary.yml
Comment (msporny) [reply vocab/credentials/v2/vocabulary.yml:142]: Changes here look fine to me (speaking as a domain expert).

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
